tsconfig.json

```diff
@@ -21,6 +21,6 @@
     "strict": true,
     "target": "es6"
   },
-  "include": ["./src/global.d.ts", "./src/**/*.ts*"],
+  "include": ["./src/global.d.ts", "./src/**/*.ts*", "src/components/show-edit-pod/show-edit-pod.stories.js"],
```

Comment:
suggestion: show-edit-pod has been deleted so this line should remove it

Reply:
agreed, this change can and should be removed now. (I'm not sure we wanted it even when the file existed tbh - do we really want to type-check a `.js` file? It's a moot question now though!)